sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -501,6 +501,7 @@ public DeletedKey deleteKey(String name) {
      * <p><strong>Code Samples</strong></p>
      * <p>Gets the deleted key from the key vault enabled for soft-delete. Prints out the details of the deleted key
      * returned in the response.</p>
+     * <pre>
      * //Assuming key is deleted on a soft-delete enabled key vault.
      * {@codesnippet com.azure.keyvault.keys.keyclient.getDeletedKey#string}
      * </pre>
```

Comment:
The opening and closing pre statements here are redundant (shortly Shawn's CheckStyle should flag these). Please remove.

Reply:
ok